validation.md

```diff
@@ -261,6 +262,22 @@ If you plan to have authorization logic in another part of your application, sim
         return true;
     }
 
+<a name="modifying-data-before-validation"></a>

```

Comment:
I guess that it should be `<a name="executing-data-before-validation"></a>` so that the anchor works